test/tools/router_check/test/config/Runtime.golden.proto.json

```diff
@@ -40,8 +50,53 @@
           "random_value": 20
         },
         "validate": {
-          "cluster_name": "www2"
+          "cluster_name": "www2",
+          "virtual_cluster_name": "",
+          "virtual_host_name": "www2",
+          "path_rewrite": "/",
+          "host_rewrite": "www.lyft.com",
+          "path_redirect": ""
+        }
+      },
+      {
+        "test_name": "Test_4",
+        "input": {
+          "authority": "www.lyft.com",
+          "path": "/disabled",
+          "method": "GET",
+          "ssl": true,
+          "internal": true,
+          "runtime": "runtime.key",
+          "random_value": 0
+        },
+        "validate": {
+          "cluster_name": "www4",
+          "virtual_cluster_name": "",
+          "virtual_host_name": "www2",
+          "path_rewrite": "/disabled",
```

Comment:
can we add a test case to demonstrate what happens when `"path": "/disabled",` and runtime is not specified?